atef/widgets/config/utils.py

```diff
@@ -373,6 +374,18 @@ def add_with_range():
             )
             self.suggest_comparison.emit(comparison)
 
+        def open_arch_viewer():
+            arch_widget = get_archive_viewer()
+            for datum in data:
+                dev_attr = '.'.join((datum.signal.parent.name, datum.attr))
+                arch_widget.add_signal(datum.pvname, dev_attr=dev_attr)
+            arch_widget.show()
+
+        menu.addSection("Open Archive Data viewer")
```

Comment:
I notice that these are hooked in to the object/component chooser specifically.

1. Any thoughts on how we could include it in the PV configurations too?
2. My intuition is that the most useful place to hook into the archiver is actually in the comparison editor- knowing that the gauge dips below x value when things are bad is most useful when you're trying to set up your greater-than comparison, for example.

Reply:
Agreed on both points. Expanding on that, perhaps:

It would be nice to have a 'view in archiver' button for the PV lists and/or context menu 'view in archiver' for individual PVs. The device configuration section could have something near the 'signals' section?
<img width="199" alt="image" src="https://user-images.githubusercontent.com/5139267/200928714-36b112c7-2527-49e4-aaff-8702e5594b54.png"> <img width="196" alt="image" src="https://user-images.githubusercontent.com/5139267/200928758-87f69450-6b40-49cf-bcfd-4532073c3ac1.png">

It'd be nice to have the option in the comparison editor itself as well, a button up near the top? A drop-down QToolButton menu for "additional actions"?
<img width="430" alt="image" src="https://user-images.githubusercontent.com/5139267/200928880-c58c931d-9286-40bb-9f68-935d23a9b16b.png">